docs/course_materials/compound_data_types.html

```diff
@@ -82,23 +83,23 @@ <h3><b>Modifying</b></h3>
                     <ul>
                         <li>Adding on a specific index:</li>
                     </ul>
-                    <pre><code data-trim data-noescape class="hljs">
+                    <pre><code data-trim data-noescape class="hljs lang-python">
 l.insert(3, 'melon')
 print(l)
 # ['orange', 'apple', 'strawberry', 'melon', 'banana']
```

Comment:
maybe for coherency, this one should be moved to the line before

Reply:
Sorry, do you mean the comment (aka output)?

Reply:
the # line, as on all other parts

Reply:
print(l)        # ['orange', 'apple', 'strawberry', 'melon', 'banana']

Reply:
Done, though there are more lines with the output on the next line. Mostly because they are long.

Should we reduce the font size / increase the width of the code box or are we ok with horizontal scrolling?

Reply:
On those cases, I think it's better to find some shorter elements :)
so strawberry can become to a cherry/pear/peach